kyo-core/shared/src/main/scala/kyo/KyoApp.scala

```diff
@@ -43,8 +41,18 @@ object KyoApp:
             this._args = args
             for proc <- initCode do proc()
 
+        protected def exit(code: Int): Unit = kernel.Platform.exit(code)
+
+        protected def onResult(result: Result[Any, Any]): Unit =
+            if !result.exists(().equals(_)) then
+                println(pprint.apply(result).plainText)
```

Comment:
Logging (JUL) instead of raw println?

Reply:
I think we can followup. I prefer that it's not a logger as it works without configuration + doesn't print `[INFO]`... or whatever.

Reply:
I may be wrong about this, but I've always thought that logger would work even without any explicit configuration.

 Generally speaking,
 * the traditional place to put diagnostic information like this on Unix has been the _Standard error output_.
 * More modern approach is to use some sort of logger, and I would expect that (unless configured otherwise) it will spill it out to STDERR as well. But the advantage is that it is more easily instrumentable.

STDOUT just didn't feel principled to me, so that's why I pointed it out in my comment above.

> we can followup

No worries. I'm writing this message to explain more my reasoning for this, not to argue. My original message was rather terse.

Reply:
My original idea was actually to make this effectful, then use the Kyo `Log` local to fetch the logger, then invoke the APIs using `Unsafe`...but they use `IO` - not general effects...Then I also realized the default slf4j logger may not output without a config file.

JUL will work without config, but it's not suited for high performance applications. Disabling a logging level does not reduce the cost significantly (if at all). It's also very expensive to translate JUL -> slf4j

Reply:
> JUL will work without config, but it's not suited for high performance applications.

> It's also very expensive to translate JUL -> slf4j

My assumption around this is that these error logs will happen very rarely. If this gets triggered only once (or a few times) per application lifetime, you don't need to worry about "high performance" or "expensive to translate". At that point, it's much more important to be interoperable.